File: clients/rust/src/systemd_manager_restart_unit_request.rs
// SystemdManagerRestartUnitRequest represents a SystemdManagerRestartUnitRequest model.
#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub struct SystemdManagerRestartUnitRequest {
    #[serde(rename="unit_name", skip_serializing_if = "Option::is_none")]
    pub unit_name: Option<String>,
}

impl SystemdManagerRestartUnitRequest {
    pub fn new(unit_name: Option<String>) -> SystemdManagerRestartUnitRequest {
        SystemdManagerRestartUnitRequest {
            unit_name,
        }
    }
}

File: clients/typescript/src/SystemdManagerRestartUnitRequest.ts

export class SystemdManagerRestartUnitRequest {
  private _unitName: string;

  constructor(input: {
    unitName: string,
  }) {
    this._unitName = input.unitName;
  }

  get unitName(): string { return this._unitName; }
  set unitName(unitName: string) { this._unitName = unitName; }
}
